SeismicMesh/geometry/signed_distance_functions.py

```diff
@@ -194,8 +194,9 @@ def eval(self, x):
 class Cylinder:
     def __init__(self, h=1.0, r=0.5):
         assert h > 0.0 or r > 0.0
```

Comment:
why `or`?

Reply:
good catch. I just opened a new one up #132. I also forget to add the `dim` field.